src/components/Settings/integrations/Organization/Modals/OrganizationAddAccountModal.tsx

```diff
@@ -153,7 +153,7 @@ export const OrganizationAddAccountModal: React.FC<
           name: yup.string().required(),
```

Comment:
Is it possible to move this schema outside of the component? I don't think it uses anything from the component closure. Having it inside the component means it will be unnecessarily recreated on every render.